src/containers/LikesDislike/LikesDislikesContainer.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import LikesDislike from "../../components/LikeAndDIslike/LikeAndDislike";
+import {
+  getLikes,
+  postLikes,
+  deleteLikes
+} from "../../redux/actions/likesDislikesActions/likesDislikesActions";
+import { ShowMessage } from "../../redux/actions/SnackBarAction";
+
+class LikesDislikeContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      like: {
+        className: "",
+        clicked: false
+      },
+      dislike: {
+        className: "",
+        clicked: false
+      }
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+  componentDidMount() {
+    const {
+      getLikes,
+
+      article: { slug }
+    } = this.props;
+    getLikes(slug);
```

Comment:
I like the fact that we can like and dislike but there is no way of knowing if I liked or disliked the article if I go to a different page and come back again.

If I liked the article and hit like again on my return visit, I get no response and both icons remain black

We can have that check in this method to check whether I liked or disliked the article then update the colours of the icons appropriately